src/test/store/publish.test.js

```diff
@@ -142,6 +147,63 @@ describe('getCheckedSharingOptions', function () {
   });
 });
 
+describe('getRemoveProfileInformation', function () {
+  it('should bail out early when there is no preference marker in the profile', function () {
+    const { getState, dispatch } = storeWithProfile();
+    // Checking to see that we don't have Preference markers.
+    expect(getHasPreferenceMarkers(getState())).toEqual(false);
+
+    // Setting includePreferenceValues option to false
+    dispatch(toggleCheckedSharingOptions('includePreferenceValues'));
+    expect(
+      getCheckedSharingOptions(getState()).includePreferenceValues
+    ).toEqual(false);
+
+    const removeProfileInformation = getRemoveProfileInformation(getState());
+    // It should return early with null value.
+    expect(removeProfileInformation).toEqual(null);
+  });
+
+  it('should remove child threads of fake main threads', function () {
```

Comment:
I checked that this test fails with the code of the main branch.

Reply:
Thanks!